### ISSUE TITLE
Add dungeon tabs/maps

### ISSUE THREAD

**Issue by AudioPixie** [AudioPixie/TPRTracker#11]
Add tabs around the outside of the map for dungeons. Would not replace the dungeon lists, but would open panels with the full map of the dungeon/area and would have chest icons for checks.

**Comment by SadieDragon** [comment]
Tl;dr- I do think this is a good idea, but I am worried about there being "too much stuff".

A concern I have here is the fact that the right side already is pretty much filled with tabs, especially with the poe/bug tabs update.
I love the tabs though!

With the spacing that the right side has, I count 9 spaces this would easily fit into- which leaves out HC unfortunately, and fills the entire left side of the map.
Spreading around the map may feel noisy, and may interfere with the settings tabs that are already there. However, it may also help new players understand what tabs mean what when they are near their dungeons- LBT, CitS, and HC are the only 3 dungeons which are in the interior, so the rest could benefit from tabs being near them on the edge.
